browser/package.json

```diff
@@ -24,17 +38,22 @@
     "glsl-rotate": "^1.1.0",
     "glslify-loader": "^2.0.0",
     "gsap": "^3.9.1",
+    "html-react-parser": "^1.4.5",
     "ipfs-http-client": "^55.0.0",
+    "node-html-parser": "^5.2.0",
     "node-polyfill-webpack-plugin": "^1.1.4",
     "raw-loader": "^4.0.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-icons": "^4.3.1",
     "react-intersection-observer": "^8.33.1",
+    "react-modal": "^3.14.4",
```

Comment:
nit: can you do this with a Tailwind modal rather than a custom library? 

https://tailwindui.com/components/application-ui/overlays/modals

Reply:
ah yeah sgtm, will update tomorrow